pkgs/development/compilers/llvm/12/default.nix

```diff
@@ -131,6 +131,7 @@ let
         echo "-B${targetLlvmLibraries.compiler-rt}/lib" >> $out/nix-support/cc-cflags
       '' + lib.optionalString (!stdenv.targetPlatform.isWasm) ''
         echo "--unwindlib=libunwind" >> $out/nix-support/cc-cflags
+        echo "-lunwind" >> $out/nix-support/cc-ldflags
```

Comment:
The previous line wasn't enough?

Reply:
Nope.

Reply:
Maybe we should make this only Android and `useLLVM`.

Reply:
Ok.